mdb-concept-recommendation.html

```diff
@@ -86,15 +86,26 @@
                 column: {
                     type: Object,
                     value: {}
+                },
+
+                isButtonClicked: {
+                    type: Boolean,
+                    value: false
                 }
             },
 
             observers: [
                 '_conceptUpdated(concept.assessment)'
             ],
 
+            listeners: {
+                'recommendation-changed-Top': '_recommendationChangedTop'
```

Comment:
Who will fire this event..??

Reply:
mdb-radio-list.html will fire 'recommendation-changed-top' event on-tap of list.